src/utils/parse-query.js

```diff
@@ -1,5 +1,12 @@
 import _ from 'lodash';
 
+const operatorSymbols = {
+  gt: '>',
+  gte: '>=',
+  lt: '<',
+  lte: '<=',
+};
```

Comment:
Can you also add `neq` to this map? https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#filter-query-parameters